src/sources/aws_kinesis_firehose/handlers.rs

```diff
@@ -128,6 +130,15 @@ pub(super) async fn firehose(
                                 path!("source_arn"),
                                 source_arn.to_owned(),
                             );
+
+                            if context.store_access_key {
+                                if let Some(access_key) = &request.access_key {
+                                    log
+                                        .metadata_mut()
+                                        .secrets_mut()
+                                        .insert_secret("access_key", access_key);
```

Comment:
```suggestion
                                        .insert_secret("aws_access_key", access_key);
```
Let's namespace this to better avoid possible naming collisions in the future.

Reply:
I'd call it `aws_kinesis_firehose_access_key` to more closely match the naming of the Splunk and Datadog tokens. `aws_access_key` makes me think of AWS IAM credentials where this is different.

Reply:
Ugh, yeah - just me confusing AWS's multitude of access keys 😵‍💫